app/helpers/katello/concerns/settings_helper_extensions.rb

```diff
@@ -15,7 +15,9 @@ def value_with_katello(setting)
           'katello_default_provision',
           'katello_default_ptable',
           'katello_default_PXELinux',
-          'katello_default_user_data'].include?(setting.name)
+          'katello_default_user_data',
+          'katello_default_kexec'
```

Comment:
Tests passed, but needed a small update to make this work - the setting needed to be in this array.